packages/x-charts/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,360 @@
+          <svg width={markerSize} height={markerSize}>
+            <path
+              // @ts-ignore TODO: Fix me
+              d={d3Symbol(d3SymbolsFill[getSymbol(shape)], markerSize)()!}
+              // TODO: Should be customizable. Maybe owner state would make more sense
+              // fill={invertMarkers ? d.stroke : d.fill}
+              // stroke={invertMarkers ? d.fill : d.stroke}
+              fill={color}
+              transform={`translate(${markerSize / 2}, ${markerSize / 2})`}
+            />
+          </svg>
+          {/* @ts-ignore */}
+          {id}: {format(series[type].series[id].data[dataIndex])}
+        </Typography>
+      ))}
+    </React.Fragment>
+  );
+}
+
+const useMouseTraker = () => {
+  const svgRef = React.useContext(SVGContext);
+
+  // Use a ref to avoid rerendering on every mousemove event.
+  const [mousePosition, setMousePosition] = React.useState<null | { x: number; y: number }>(null);
+
+  const handleMouseOut = React.useCallback(() => {
+    setMousePosition(null);
+  }, [setMousePosition]);
+
+  const handleMouseMove = React.useCallback(
+    (event: MouseEvent) => {
+      setMousePosition({
+        x: event.clientX,
+        y: event.clientY,
+      });
+    },
+    [setMousePosition],
+  );
+
+  React.useEffect(() => {
+    const element = svgRef.current;
+    if (element === null) {
+      return () => {};
+    }
+
+    element.addEventListener('mouseout', handleMouseOut);
+    element.addEventListener('mousemove', handleMouseMove);
+    return () => {
+      element.removeEventListener('mouseout', handleMouseOut);
+      element.removeEventListener('mousemove', handleMouseMove);
+    };
+  }, [handleMouseMove, handleMouseOut, svgRef]);
+
+  return mousePosition;
+};
+
+export type TooltipProps = {
+  /**
+   * Select the kinf of tooltip to display
+   * - 'item': Shows data about the item below the mouse.
+   * - 'axis': Shows vlaues associated to the hovered x value
+   * - 'none': Does nto display tooltip
+   * @default 'item'
+   */
+  trigger?: 'item' | 'axis' | 'none';
+  /**
+   * Props propagate to the highlight
+   */
+  highlightProps?: Partial<HighlightProps>;
+};
+
+export function Tooltip(props: TooltipProps) {
+  const { trigger = 'axis', highlightProps } = props;
+
+  useAxisEvents(trigger);
+  const mousePosition = useMouseTraker();
+
+  const { item, axis } = React.useContext(InteractionContext);
+
+  const highlightRef = React.useRef<SVGPathElement>(null);
+
+  const displayedData = trigger === 'item' ? item : axis;
+
+  const popperOpen =
+    mousePosition !== null &&
+    ((trigger === 'item' && displayedData !== null) ||
+      (trigger === 'axis' &&
+        ((displayedData as AxisInteractionData).x !== null ||
+          (displayedData as AxisInteractionData).y !== null)));
+
+  if (trigger === 'none') {
+    return null;
+  }
+  return (
+    <NoSsr>
+      <React.Fragment>
```

Comment:
Is the `React.Fragment` needed here ? I don't see why `NoSsr` would not support multiple child by looking at its codebase